pyincore/analyses/meandamage/meandamage.py

```diff
@@ -173,10 +173,10 @@ def mean_damage_bulk_input(self, damage, dmg_ratio_tbl):
         return result
 
     def mean_damage(self, dmg, dmg_ratio_tbl, damage_interval_keys, is_bridge):
-        """Calculates building damage results for a single building.
+        """Calculates mean damage based on damage probabilities and damage ratios
 
         Args:
-            dmg (obj): dmg analysis output for a single entry.
+            dmg (obj): dmg analysis output for a single building.
```

Comment:
This is generic so it shouldn't say for a building. Maybe "damage analysis output for a single entity in the built environment"?

Reply:
ah right! fixed. thanks for pointing it out